README.md

```diff
@@ -1,5 +1,136 @@
+- Supports multiple entry points
+- [Generate stub entry points](#crackle-dev) for local development
+- [ESM reconciliation](#esm-reconciliation)
+- [Handles side-effects](#side-effects) for generated bundles
+- Compile Vanilla Extract stylesheets to JavaScript
+- 🚧 Start an HTTP server (with hot reloading) to preview the website **(WIP)**
+- 🚧 Build a static version the site (e.g. for deploying to S3) **(WIP)**
+
+## Contents
+
+- [Features](#features)
+- [Contents](#contents)
+- [Getting started](#getting-started)
+  - [`crackle package`](#crackle-package)
+    - [Externals](#externals)
+  - [`crackle fix`](#crackle-fix)
+  - [`crackle dev`](#crackle-dev)
+- [Side-effects](#side-effects)
+- [ESM reconciliation](#esm-reconciliation)
+- [DTS bundles](#dts-bundles)
+- [Backwards-compatible entry points](#backwards-compatible-entry-points)
+- [Contributing](#contributing)
+  - [Self-hosting / Bootstrap](#self-hosting--bootstrap)
+
+## Getting started
+
+```sh
+pnpm install --dev @crackle/cli
+```
+
+Crackle will look for a `crackle.config.ts` file in the project directory, but if one doesn't exist, it will use the default configuration.
+
+Here's how to specify a custom configuration file:
+
+```ts
+// crackle.config.ts
+import type { CrackleConfig } from '@crackle/core';
+
+export default {
+  // ...
+} satisfies CrackleConfig;
+```
+
+You can find the full list of commands by running `crackle --help`.
+Here are the most common ones:
+
+### `crackle package`
+
+Compile the package for publishing.
+This will compile TypeScript files to CJS- and ESM-compatible files and generate TypeScript declaration files (`.d.ts` files) for entry points.
+
+The default entry point is `src/index.ts`.
+Multiple (nested) entry points are supported, by adding `.ts` files to `src/entries/`.
+Crackle will also run [`crackle fix`](#crackle-fix) if passed the `--fix` parameter.
+
+#### Externals
+
+If a dependency is present in `devDependencies` (but not in `peerDependencies`) it bundled with the project's source code.
+`dependencies`, `peerDependencies` and `optionalDependencies` are marked as external and not bundled.
+
+### `crackle fix`
+
+Updates `package.json` exports, files, [`sideEffects` field](#side-effects) and more:
+
+- `package.json`: `"main"`, `"module"` and `"types"` are updated to point to the generated files in `dist`
+- `package.json`: `"exports"` and `"files"` are updated to include generated entry points
+- `package.json`: [`"sideEffects"` flag](#side-effects) is updated to match the generated files in `dist`
+- `package.json`: keys are ordered to a standard order using [`sort-package-json`](https://github.com/keithamus/sort-package-json)
+- `.gitignore` is updated to ignore [backwards-compatible entry points](#backwards-compatible-entry-points)
+
+### `crackle dev`
+
+Generate entry points for local development.
+This will generate stub entry points for local development, which will import the source files directly instead of the compiled files.
```

Comment:
```suggestion
This will generate stub entry points for local development.
Stub entry points import the source files directly instead of the compiled files.
```